docs/changelog.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## v0.2 (unreleased)
```

Comment:
We should add date

Reply:
yeah, will do when just before the release